packages/storybook-html/src/components/core/colour-base.story.mdx

```diff
@@ -35,21 +35,21 @@ These are the core brand colours for the University of Queensland. Refer to the
     </tr>
     <tr  className="uqds-table__row">
       <td className="uqds-table__item"><div className="uqds-colour__palette uqds-colour--purple"></div></td>
-      <td className="uqds-table__item" data-title="Name:">UQ purple</td>
+      <td className="uqds-table__item" data-title="Name:">UQ Purple</td>
```

Comment:
Revert the naming. Should be sentence casing. UQ purple is not really a proper noun.

Reply:
Done